Sources/Purchasing/Purchases/PurchaseParams.swift

```diff
@@ -89,6 +92,23 @@ import Foundation
             return self
         }
 
+        #if !ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION
```

Comment:
hmmmm wondering if it might make sense to gate the whole PurchaseParams API behind this flag 🤔

Reply:
It wouldn't be a bad idea since we usually try to keep the API surface area for `ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION` as small as possible